content/en/docs/concepts/configuration/secret.md

```diff
@@ -672,6 +672,35 @@ A container using a Secret as a
 Secret updates.
 {{< /note >}}
 
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+For clusters that are extensively using secrets (having at least tens of thousands of unique
+Secret to Pod mounts), Kubernetes has an _Immutable Secrets/ConfigMaps_ feature. It allows you
+to mark individual Secrets/ConfigMaps as `Immutable`, which blocks any changes to their data
+and in return:
```

Comment:
Can I use an immutable Secret if I only have a few hundred Secrets in my cluster?

Reply:
Yes - but I don't see where the text suggests you can't.

Reply:
> For clusters that are extensively using secrets (having at least tens of thousands of unique Secret to Pod mounts), Kubernetes has an _Immutable Secrets/ConfigMaps_ feature.

This could read like a requirement.

Reply:
What do you suggest? [I don't read it as a requirement]